skl2onnx/operator_converters/label_binariser.py

```diff
@@ -14,10 +14,9 @@ def convert_sklearn_label_binariser(scope, operator, container):
     """Converts Scikit Label Binariser model to onnx format."""
     binariser_op = operator.raw_operator
     classes = binariser_op.classes_
-    input_samples = operator.inputs[0].type.shape[0]
-    zeros_tensor = np.full((input_samples, len(classes)),
+    zeros_tensor = np.full((1, len(classes)),
                            binariser_op.neg_label, dtype=np.float)
-    unit_tensor = np.full((input_samples, len(classes)),
+    unit_tensor = np.full((1, len(classes)),
```

Comment:
Why these changes, are we not planning on supporting batch prediction?

Reply:
Yes we do. It is using broadcasting. We cannot use input_samples in any converter because it is not known at conversion time. I replaced by 1 because we probably tested this converter either with the same number of samples or with we specifiied 1 input sample at conversion time.